src/test/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicatorTest.java

```diff
@@ -38,6 +38,7 @@
 import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.SlowShutterStatus;
 import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.WBMode;
 import com.avispl.symphony.dal.communicator.aver.ptz.interfaces.DevelopmentTest;
+import com.avispl.symphony.dal.communicator.aver.ptz.interfaces.ProductionTest;
```

Comment:
instead of production and development, can we clarify it to read device and mock name?